bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -48,6 +48,44 @@ spec:
         - |
           #!/bin/bash
           set -exuo pipefail
+{{ if .NETWORK_IDENTITY_ENABLE }}
```

Comment:
Prior to this feature and without it enabled it will use the service account.

Reply:
OK, I guess I am a bit confused here because if this is just to use the `kubectl get csr ...` some lines below, I don't know why we would use the node identity credentials instead of just keep using the service account ones. Is it again just because we are replacing the service account from the role bindings? I guess here is not as bad, as this container is already priviledged?

Reply:
It is also creating the CSR, not only getting it.
I think it is reasonable to use the per-node cert in this case, especially because the container is already privileged.